client/src/components/ExchangeList.tsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback, useMemo, useRef } from 'react'
+import { AgGridReact } from 'ag-grid-react'
+import 'ag-grid-community/styles/ag-grid.css'
+import 'ag-grid-community/styles/ag-theme-alpine.css'
```

Comment:
These css typical added on the main app.tsx only
not sure why they ended up everywhere, but no big deal we can take care of that later

Reply:
just a copy paste thing, removed now :)